### ISSUE TITLE
Non-admins can't move navigation items

### ISSUE THREAD

**Issue by MoritzLost** [verbb/navigation#301]
**Description**

We're seeing a bug in the current version of the plugin for Craft 4. Every user without admin privileges can't move navigation nodes. For admins, the little arrow icon shows up normally and can be used to drag and drop nodes to a new position:

![navigation-admin](https://user-images.githubusercontent.com/10146880/170219097-74176ce7-bd58-4103-9a61-1ef215d489da.png)

But for editors, the move handles are missing:

![navigation-editor](https://user-images.githubusercontent.com/10146880/170219128-9615976c-6b91-4a4f-9739-703b91fddb94.png)

Those editors have the permission to edit the navigation in question (`navigation-manageNav:[uid]`), but not to create new navigations or edit navigation settings or delete navigations.

**Steps to reproduce**
1. Create a navigation.
2. Log in as a non-admin user with permission to edit the navigation (but without permissions to create new navigations, edit navigation settings or delete navigations).
3. Go to edit the navigation and try to move a node to a new permission.

**Additional info**

- Plugin version: 2.0.1
- Craft version: 4.0.3
- Multi-site: No

**Additional context**

- PHP 8.1

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/navigation` requirement in `composer.json` to:

```
"require": {
  "verbb/navigation": "dev-craft-4 as 2.0.1",
  "...": "..."
}
```

Then run `composer update`.

**Comment by MoritzLost** [comment]
@engram-design That was fast, thanks! I've tested the dev branch and it solves the issue indeed. Looking forward to the next release!

**Comment by engram-design** [comment]
Fixed in [2.0.2](https://github.com/verbb/navigation/releases/tag/2.0.2)
